meeteval/wer/matching/cy_levenshtein.pyx

```diff
@@ -176,18 +124,17 @@ def time_constrained_levenshtein_distance(
         cost_ins: uint = 1,
         cost_sub: uint = 1,
         cost_cor: uint = 0,
+        prune: bool = True
```

Comment:
What does `time_constrained_levenshtein_distance_v2_`, when prune is True, while it should be False?
If the result would be wrong, I suggest two changes:
 - Change the default to False or None (None would mean calculate, whether prune can be used, not sure about the overhead)
 - Add a comment, that no user should set prune to True, it will be computed.

Reply:
Added a function that computes whether pruning is applicable and set the default to `'auto'`